dist/pom.xml

```diff
@@ -49,28 +49,6 @@
         <rapids.source.jar.phase>none</rapids.source.jar.phase>
     </properties>
     <profiles>
-        <profile>
```

Comment:
@NvTimLiu can you please double-check if there are any other references to these profiles? There might be some outside of this repository, thanks